mu-plugins/blocks/query-filter/render.php

```diff
@@ -0,0 +1,127 @@
+	data-wp-on--mouseenter="actions.wporg.queryFilter.handleMouseEnter"
+	data-wp-on--mouseleave="actions.wporg.queryFilter.handleMouseLeave"
+	data-wp-effect="effects.wporg.queryFilter.init"
+	data-wp-class--is-modal-open="context.wporg.queryFilter.isOpen"
+>
+	<button
+		class="wporg-query-filter__toggle <?php echo count( $settings['selected'] ) ? 'has-filter-applied' : 'has-no-filter-applied'; ?>"
+		data-wp-class--is-active="context.wporg.queryFilter.isOpen"
+		data-wp-on--click="actions.wporg.queryFilter.toggle"
+		data-wp-bind--aria-expanded="context.wporg.queryFilter.isOpen"
+		aria-controls="<?php echo esc_attr( $html_id ); ?>"
+	><?php echo wp_kses_post( $settings['label'] ); ?></button>
+
+	<div
+		class="wporg-query-filter__modal-backdrop"
+		data-wp-bind--hidden="!context.wporg.queryFilter.isOpen"
+		data-wp-on--click="actions.wporg.queryFilter.toggle"
+	></div>
+
+	<div
+		class="wporg-query-filter__modal"
+		id="<?php echo esc_attr( $html_id ); ?>"
+		data-wp-bind--hidden="!context.wporg.queryFilter.isOpen"
+		data-wp-effect="effects.wporg.queryFilter.focusFirstElement"
+		data-wp-on--keydown="actions.wporg.queryFilter.handleKeydown"
+	>
+		<form
+			action="<?php echo esc_attr( $settings['action'] ); ?>"
+			data-wp-on--change="actions.wporg.queryFilter.handleFormChange"
+		>
+			<div class="wporg-query-filter__modal-header">
+				<h2><?php echo wp_kses_post( $settings['label'] ); ?></h2>
+				<input
+					type="button"
+					class="wporg-query-filter__modal-close"
+					data-wp-on--click="actions.wporg.queryFilter.toggle"
+					aria-label="<?php esc_attr_e( 'Close', 'wporg' ); ?>"
+				/>
+			</div> <!-- /.wporg-query-filter__modal-header -->
+			<div class="wporg-query-filter__modal-content">
+				<?php foreach ( $settings['options'] as $value => $label ) : ?>
+				<div class="wporg-query-filter__option">
+					<input
+						type="checkbox"
+						name="<?php echo esc_attr( $settings['key'] ); ?>[]"
+						value="<?php echo esc_attr( $value ); ?>"
+						id="<?php echo esc_attr( $html_id . '-' . $value ); ?>"
+						<?php checked( in_array( $value, $settings['selected'] ) ); ?>
+					/>
+					<label for="<?php echo esc_attr( $html_id . '-' . $value ); ?>"><?php echo esc_html( $label ); ?></label>
+				</div>
+				<?php endforeach; ?>
+			</div>
+
+			<?php
+			/**
+			 * Fires inside the filter form, right before action buttons.
+			 *
+			 * @param string   $key   The key for the current filter.
+			 * @param WP_Block $block The current block being rendered.
+			 */
+			do_action( 'wporg_query_filter_in_form', $settings['key'], $block );
+			?>
+
+			<div class="wporg-query-filter__modal-actions">
+				<input
+					type="button"
+					class="wporg-query-filter__modal-action-clear"
+					value="<?php esc_attr_e( 'Clear', 'wporg' ); ?>"
+					data-wp-on--click="actions.wporg.queryFilter.clearSelection"
+				/>
+				<input
+					type="submit"
+					value="<?php esc_html_e( 'Apply', 'wporg' ); ?>"
```

Comment:
I think it was agreed that this would be updated with the filter count:

[Context Link](https://github.com/WordPress/wporg-mu-plugins/issues/419#issuecomment-1679390461)
[Figma Link](https://www.figma.com/file/HOJTpCFfa3tR0EccUlu0CM/WordPress.org-Design-Library?type=design&node-id=2847-4389&mode=design&t=Y2B54ZHq9soTX1nE-0)

| Image |
| --- |
| <img width="200" alt="Screenshot 2023-09-06 at 10 22 42 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/1657336/e04fdba4-676d-4c33-99e9-874a549ae778"> |

Reply:
I've been only referencing the "filters 6.1" page in the Filter component file, thinking that was the final version 🙃 

The link doesn't work for me (opens the file but wrong page)… but I found it by searching through the other pages, so thanks for pointing it out.

@WordPress/meta-design — Can you confirm how this should behave? It sounds like the top number (on black, red highlight) would be the number of filters currently applied on the page, while the Apply button (on blue, yellow highlight) would be the number of filters currently selected— so it would stay "Popular tags 1" even if you select two more, but the button would be "Apply (3)"?

<img width="542" alt="Screenshot 2023-09-06 at 10 50 06 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/541093/f9e03b81-4938-446d-932f-a5e2cbb93d64">

Reply:
> It sounds like the top number (on black, red highlight) would be the number of filters currently applied on the page, while the Apply button (on blue, yellow highlight) would be the number of filters currently selected— so it would stay "Popular tags 1" even if you select two more, but the button would be "Apply (3)"?

Correct, but honestly I think we can remove the number on the Apply button for now, I'm not sure it adds clarity. If Francisco when he comes back from AFK has to correct me, I will buy him an apology milkshake at that time. How does this sound?

Reply:
I think it was an accessibility request, but maybe it would be easier to understand the UX once this is merged and live on the staging site. I'll leave this as-is (no number on button) for this initial pass, and ask for feedback once there's a live place to try it out.

Reply:
@ryelle is correct. It was an accessibility request shared [here](https://github.com/WordPress/wporg-mu-plugins/issues/419#issuecomment-1679390461).